### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by omahs**
Fix typos

### CODE REVIEW

File: src/diffusers/pipelines/pag/pag_utils.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def get_block_type(module_name):
         def get_block_index(module_name):
```
Comment (a-r-r-o-w) [inline src/diffusers/pipelines/pag/pag_utils.py:84]: Could you revert the changes here so I don't have a merge conflict to resolve in #8936? :sweat_smile: It's taken care of in the PAG refactor

**Review by a-r-r-o-w — APPROVED** [review]
Thanks, looks good!

File: src/diffusers/pipelines/pag/pag_utils.py (hunk 1 above)
Comment (omahs) [reply src/diffusers/pipelines/pag/pag_utils.py]: @a-r-r-o-w Hey, sure! Just reverted the changes

**Comment by yiyixuxu** [comment]
thank you!
